omou-frontend/src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -291,123 +292,119 @@ class Scheduler extends Component {
-                                    <SearchIcon />
-                                </IconButton>
-                            </Grid>
-                            <Grid item md={1} lg={1}>
-                                {(this.state.calendarIcon) ?
-                                    <FormControl className={'filter-select'}>
-                                        <InputLabel htmlFor="filter-class-type"></InputLabel>
-
-                                        <Select
-                                            native
-                                            value={this.state.filterValue}
-                                            onChange={this.handleFilterChange('filterValue')}
-                                            inputProps={{
-                                                name: 'filterValue',
-                                                id: 'filter-class-type',
-                                            }}
-                                        >
-                                            <option value={"C"}>Class</option>
-                                            <option value={"T"}>Tutor</option>
-                                        </Select>
-                                    </FormControl>
-                                    :
-                                    <FormControl className={'filter-select'}>
-                                        <InputLabel htmlFor="filter-resource-type"></InputLabel>
-
-                                        <Select
-                                            native
-                                            value={this.state.resourceFilterValue}
-                                            onChange={this.handleResourceFilterChange('resourceFilterValue')}
-                                            inputProps={{
-                                                name: 'resourceFilterValue',
-                                                id: 'filter-resource-type',
-                                            }}
-                                        >
-                                            <option value={"R"}>Room</option>
-                                            <option value={"I"}>Instructors</option>
-                                        </Select>
-                                    </FormControl>
-                                }
+                <Paper className="paper" style={{ "padding": "2%" }}>
+                    <Typography variant="h3" align="left">Scheduler</Typography>
+                    <br />
+                    <Grid container
+                        direction="row"
+                        alignItems="center"
+                        className="scheduler-header">
+                        <Grid item className={"calendar-icon-header"}>
+                            <IconButton color={this.state.calendarIcon ? "primary" : ""} onClick={this.changeViewToCalendar} className={'calendar-icon'} aria-label='next-month'>
```

Comment:
Break up longer lines like these into multiple lines (usually between props), it's often easier to read multiple short lines than one long line.